app/src/main/java/fr/free/nrw/commons/nearby/fragments/NearbyParentFragment.java

```diff
@@ -626,6 +627,11 @@ public void centerMapToPlace(Place place) {
     @Override
     public void updateListFragment(List<Place> placeList) {
         adapterFactory.updateAdapterData(placeList, (RVRendererAdapter<Place>) rvNearbyList.getAdapter());
+        if (placeList.size() <= 0) {
+            noResultsView.setVisibility(View.VISIBLE);
```

Comment:
A ternary would look nice here and avoid repeating `noResultsView.setVisibility(`

Reply:
That does look much better! 👍